py/DREAM/Settings/Equations/RunawayElectrons.py

```diff
@@ -229,6 +254,12 @@ def fromdict(self, data):
                 self.comptonPhotonFlux  = data['compton']['flux']
                 self.comptonPhotonFlux_t = np.array([0.0])
 
+        if 'C1' in data['compton']:
+            self.C1_Compton = data['compton']['C1']
+            self.C2_Compton = data['compton']['C2']
+            self.C3_Compton = data['compton']['C3']
+            self.integratedComptonSpectrum = quad(GammafluxProfil, 0, np.inf, args=(self.C1_Compton, self.C2_Compton, self.C3_Compton))[0]
```

Comment:
It would feel better if this was done in the C++ code (since anything could happen with this number between being calculated and actually used in the kernel), but I can see that this is more convenient for the programmer, and so maybe it's okay...